### DISCUSSION TITLE
Unable to use SSL when communicating with `MySQL`

### DISCUSSION THREAD

**Discussion by abhi-markan** [keystonejs/keystone#9009] [Questions]
Unable to use `?sslcert=./DigiCertGlobalRootCA.crt.pem` when using TLS with MySQL.
As per Prisma it should be in `./prisma` directory which does not exists in keystone so placed the CA cert in root.

https://www.prisma.io/docs/concepts/components/prisma-schema/data-sources#securing-database-connections

MySQL connection over TLS.

**Comment by stuible** [comment]
I am also encountering this issue.  Would love to be able to use SSL with MySQL.

**Comment by ttbarnes** [comment]
@dcousens Would it be straight forward to move the `.crt.pem`into the `./prisma` directory? 🙏

**Comment by dcousens** [comment]
Maybe an alternative is for us to build an example where we have a custom `prismaClientPath: './.myprisma/client'`, and then have a TLS certificate in there?

> @dcousens Would it be straight forward to move the .crt.peminto the ./prisma directory? 🙏

It doesn't feel right to me for something to automatically copy your `.pem` certificate anywhere, and especially not to your `node_modules` if that is where the Prisma client is.
